scenarios/kubernetes_kops_aws.py

```diff
@@ -78,9 +78,10 @@ def cluster_name(cluster, build):
     """Return or select a cluster name."""
     if cluster:
         return cluster
+    domain = 'test-aws.k8s.io'
     if len(build) < 20:
-        return 'e2e-%s' % build
-    return 'e2e-%s' % hashlib.md5(build).hexdigest()[:10]
+        return 'e2e-%s.%s' % (build, domain)
+    return 'e2e-%s.%s' % (hashlib.md5(build).hexdigest()[:10], domain)
```

Comment:
Did you intend to do `:10` but check if `len(build) < 20` - it feels like they should be the same?